packages/dev/serializers/src/glTF/2.0/glTFAnimation.ts

```diff
@@ -263,12 +263,16 @@ export class _GLTFAnimation {
         bufferViews: IBufferView[],
         accessors: IAccessor[],
         convertToRightHandedSystem: boolean,
-        animationSampleRate: number
+        animationSampleRate: number,
+        shouldExportAnimation?: (animation: Animation) => boolean

```

Comment:
`shouldExportAnimation` sounds more like a boolean than a predicate... Maybe `exportAnimationPredicate` or `animationPredicate` instead?

Reply:
We already have `shouldExportNode` on the options that's also a function, so I kept the same pattern.

Reply:
I think it could be confusing if we have two schemes, if we could deprecate the shouldExportNode name and set a new basis for both names it would be nicer. But I'm not sure how we would proceed in deprecating the shouldExportNode name? Just mark it as so and then add a new property with the new name? 🤔

Reply:
let's just keep it this way. Ready to merge.